torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -59,13 +59,13 @@ def _put_into_sample(self, sample: Any, id: int, item: Any) -> Any:
 
     def _apply_image_transform(
         self,
-        image: Union[torch.Tensor, PIL.Image.Image, features.Image],
+        image: ImageType,
         transform_id: str,
         magnitude: float,
         interpolation: InterpolationMode,
-        fill: Union[Dict[Type, FillType], Dict[Type, None]],
-    ) -> Any:
-        fill_ = fill[type(image)]
+        fill: Optional[Dict[Type, Optional[FillType]]],
+    ) -> ImageType:
+        fill_ = fill[type(image)] if fill is not None else None
```

Comment:
Restoring @vfdev-5's earlier approach due to the new definition of the input. We adopt the same idiom elsewhere.